manifests/mod/version.pp

```diff
@@ -0,0 +1,12 @@
+# @summary
+#   Installs `mod_version`.
+#
+# @see https://httpd.apache.org/docs/current/mod/mod_version.html for additional documentation.
+#
+class apache::mod::version {
+  if $facts['os']['family'] == 'debian' {
+    warning("${module_name}: module version_module is built-in and can't be loaded")
+  } else {
+    ::apache::mod { 'version': }
```

Comment:
The leading `::` is not needed. AFAIK it never was for defines, but certainly since we dropped Puppet < 4 it's no longer needed.